### DISCUSSION TITLE
Adding SHIFT + OPTION + <Number> for throwing focused window to space is not working.

### DISCUSSION THREAD

**Discussion by adilk0728** [ianyh/Amethyst#1673] [Q&A]
Hey! 

I am on version `0.21.1` and I have retained the default for `mod1` as `shift` + `option`.  I am using a config file and running MacOS Sonoma 14.

Now I want to use `mod1` + `<number>`   to **Throw focused window to space** but when I do that it either adds some accented characters or does not do anything.

Now I have researched a bit and tried changing the Keyboard to U.S., ABC, UNICODE HEX but for no avail.

I don't want to remap it to something different cause it works on my personal laptop.

Any suggestions would be helpful.

**Comment by ianyh** [comment]
See https://github.com/ianyh/Amethyst/discussions/1682
